snuba/datasets/__init__.py

```diff
@@ -22,6 +22,19 @@ def get_schema(self):
     def get_processor(self):
         return self.__processor
 
+    def get_writer(self, options=None):
+        from snuba.clickhouse import ClickhousePool
+        from snuba.writer import NativeDriverBatchWriter
```

Comment:
These are to escape a circular import due to `snuba.datasets` being imported regularly in other parts of the codebase. We should find a way to restructure this later to avoid.